inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/graph/RdfUtils.java

```diff
@@ -77,12 +90,27 @@
             throws QueryEvaluationException {
         return conn.getStatements(subj, pred, obj, includeInferred);
     }
-    
+
     public static RepositoryResult<Statement> getStatementsSparql(RepositoryConnection conn,
-            Resource subj, IRI pred, Value obj, boolean includeInferred)
+        Resource subj, IRI pred, Value obj, boolean includeInferred)
```

Comment:
Can we change `RepositoryResult` also to `Optional` here instead of passing it up the call stack?

Reply:
Correct